packages/dashboard/package.json

```diff
@@ -36,8 +36,8 @@
     "@storybook/addon-essentials": "^6.5.12",
     "@storybook/addon-interactions": "^6.5.12",
     "@storybook/addon-links": "^6.5.12",
-    "@storybook/builder-webpack5": "^6.5.12",
-    "@storybook/manager-webpack5": "^6.5.12",
+    "@storybook/builder-webpack4": "^6.5.13",
```

Comment:
Should we upgrade other storybook packages to `^6.5.13` as well so they stay in sync?